compiler/src/iree/compiler/Codegen/LLVMGPU/test/ROCDL/config_vector_distribute_reduction_gfx942.mlir

```diff
@@ -291,3 +291,80 @@ func.func @test_gather_config(%arg0: !iree_tensor_ext.dispatch.tensor<readonly:t
 // CHECK-SAME:               subgroup_basis = {{\[}}[1, 1], [0, 1]],
 // CHECK-SAME:               thread = [0, 1], thread_basis = {{\[}}[1, 64], [0, 1]],
 // CHECK-SAME:               workgroup = [1, 0]
+
+// -----
+
+#map = affine_map<()[s0] -> (s0 floordiv 4)>
+#map1 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+#map3 = affine_map<(d0, d1, d2) -> (d2)>
+#pipeline_layout = #hal.pipeline.layout<constants = 6, bindings = [#hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, "ReadOnly|Indirect">, #hal.pipeline.binding<storage_buffer, Indirect>, #hal.pipeline.binding<storage_buffer, Indirect>], flags = Indirect>
+module {
+  func.func @test_store_to_buffer(%arg0: index, %arg1: index) {
```

Comment:
Can you make the test case simpler? I think you only need few ops

Reply:
I tried, but when I removed one of the linalg generics, it goes to the TileAndFuse pipeline.

Reply:
I think you can remove the last linalg.generic (i.e., with only parallel iterators).